src/span.js

```diff
@@ -64,6 +66,10 @@ export default class Span {
         return this._operationName;
     }
 
+    get finished(): boolean {
```

Comment:
why does this have to be public? do not extend the public interface unless you have to - much easier to add later than to remove.